tests/sdk/test_debug.py

```diff
@@ -15,3 +17,49 @@ def test_client_multi_nodes_dataset(dataset_query):
     key = client.add_dataset(dataset_query)
     asset = client.get_dataset(key)
     assert asset.owner == 'owner_1'
+
+
+def test_client_multi_nodes_cp(asset_factory):
+    """Assert that there is one CP local folder per node"""
+    client = substra.Client(debug=True)
+
+    dataset_query = asset_factory.create_dataset()
+    dataset_query.metadata = {substra.DEBUG_OWNER: 'owner_1'}
+    dataset_key = client.add_dataset(dataset_query)
```

Comment:
Could you rename the variable dataset_1_key (similar to what you've done with dataset_2_key) ?